io/eolearn/io/extra/meteoblue.py

```diff
@@ -291,10 +292,10 @@ def _meteoblue_timestamps_from_time_interval(timestamp_pb):
     # Regular time intervals return `start, end and stride` as a time axis
     # We convert it into an array of daytime
     time_range = range(timestamp_pb.start, timestamp_pb.end, timestamp_pb.stride)
-    return list(map(dt.date.fromtimestamp, time_range))
+    return list(map(dt.date.fromtimestamp, time_range))  # type:ignore[arg-type]
```

Comment:
I guess the issue here is that the function wants to return `List[dt.datetime]`, but you are using `dt.date.fromtimestamp`, which returns a `date` object.

Should work if you use 

```suggestion
    return list(map(dt.datetime.fromtimestamp, time_range))  # type:ignore[arg-type]
```

Reply:
ah, I scanned the error and thought it had some discrepancy between integers and float.

do we consider it safe to change this functionality?

Reply:
Actually, you may be right.. it's used in dataframes, where we usually do work with dates, but the code uses the `timestamps` variables, sam as we do in area-monitoring, but in fact we always limit the timestamps up to the date point. 

Another option would be to change the return type, but I don't think it's problematic in any case.

Reply:
tests pass with this change, so i'm doing it